homeassistant/helpers/translation.py

```diff
@@ -258,17 +264,20 @@ async def _async_load(self, language: str, components: set[str]) -> None:
         _LOGGER.debug(
             "Cache miss for %s: %s",
             language,
-            ", ".join(components),
+            components,
         )
         # Fetch the English resources, as a fallback for missing keys
         languages = [LOCALE_EN] if language == LOCALE_EN else [LOCALE_EN, language]
 
         integrations: dict[str, Integration] = {}
-        domains = list({loaded.partition(".")[0] for loaded in components})
+        domains = {loaded.partition(".")[0] for loaded in components}
```

Comment:
The conversion to list was not needed since `async_get_interations` takes an iterable